### DISCUSSION TITLE
Missing enhancement from PR #625

### DISCUSSION THREAD

**Discussion by devMaFi** [docToolchain/docToolchain#859] [Q&A]
Hello,

thanks for the great work around, this is helping us out a lot in our day to day work.

Anyhow, I just want to check if I missed something but when migrating to version 2 in our environment I was not able to found the enhancement from PR #625

Am I missing something?

Thanks Martin

**Comment by rdmueller** [comment]
thanx for the feedback. As I see, the PR #625 was merged to the master branch but most of the development currently takes place on the ng-branch.

There is a PR for merging the master into the ng-branch and it seems that your changes are currently stuck there: https://github.com/docToolchain/docToolchain/pull/824

I see currently two ways how to progress from here:

1) I need to find some time to check PR #824 
2) you create a new PR with your changes to merge with the ng-branch

I'll put 1) on my todo list, but can't promise any timeline for that

**Reply by devMaFi** [reply]
Hi, no worries, I create a new PR to be merged into the ng-branch.

**Reply by devMaFi** [reply]
I have created a new pull request on the NG branch: #860
In case of questions, I'm happy to help!

Best Martin

**Reply by rdmueller** [reply]
merged
